src/comps/Header.tsx

```diff
@@ -22,9 +22,8 @@ import Metrics from "./ui/metrics";
 // converting to lower case to avoid case sensitive issue
 
 const Header = ({ config }: { config: BridgeConfig }) => {
-  const isTestnet =
-    config.WALLET_NETWORK?.toLowerCase() === "sbtcTestnet".toLowerCase();
-
+  //const isTestnet = config.WALLET_NETWORK?.toLowerCase() === "sbtcTestnet".toLowerCase();
+  const isTestnet = true;
```

Comment:
This is for the faucet it will only work in the sbtcTestnet revert back to the way it was

Reply:
Reverted back